iam/snippets/src/test/java/RoleIT.java

```diff
@@ -111,6 +111,10 @@ public void testRole() throws IOException {
     ListRoles.listRoles(projectId);
     assertThat(bout.toString().contains(roleId));
 
+    // Test disable role.
+    DisableRole.disableRole(projectId, roleId);
+    assertThat(bout.toString().contains("Stage: DISABLED"));
```

Comment:
Instead of asserting from sysout, it's more robust to retrieve and verify the status from the updated role.